### ISSUE TITLE
Use java.io.File to concatenate path components

### ISSUE THREAD

**Issue by rslemos** [OpenLiberty/liberty-arquillian#104]
When concatenating path components, care should be taken to introduce path separators ("/") when needed.

In:
https://github.com/OpenLiberty/liberty-arquillian/blob/f25954c0475a7760a51960f1ad5387e1ec5833a7/liberty-managed/src/main/java/io/openliberty/arquillian/managed/WLPManagedContainer.java#L1430

it should be noted that `getWlpUsrDir()` must return a directory with `/` appended to it.

Otherwise it may error out with:
```
Caused by: java.io.FileNotFoundException: C:\dev-env\wlp-usrservers\myserver\server.xml (The system could not find the specified path.)
```

**Comment by cherylking** [comment]
This was actually fixed in this PR last year: https://github.com/OpenLiberty/liberty-arquillian/pull/70

We just have not released version 1.0.7 yet with the fix. I will work on that and post here when it is available.

**Comment by cherylking** [comment]
Release 1.0.7 is available with the fix for this issue: https://github.com/OpenLiberty/liberty-arquillian/releases/tag/arquillian-parent-liberty-1.0.7
